charts/testkube-cloud-api/templates/ingress-ws.yaml

```diff
@@ -24,6 +24,8 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-body-size: 8m
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    nginx.org/websocket-services:  {{ include "testkube-cloud-api.fullname" . }}
```

Comment:
This is an annotation for the NGINX Ingress Controller from nginx.org, which we don't use.

We use this one https://github.com/kubernetes/ingress-nginx which does not support this annotation